react/modules/analytics/utils.ts

```diff
@@ -28,12 +28,17 @@ export const analyticsURLParams = [
 
 export function shouldInvalidateCurrentCampaign(
   location: Location,
-  referrer: string
+  referrer: string,
+  storageReferrer: string
 ) {
   const referrerURL = referrer ? new URL(referrer) : null
 
   // if user comes from a referring website
-  if (referrerURL && referrerURL.host !== location.host) {
+  if (
```

Comment:
What do you think about commenting above about the need to compare the referrer with the storageReferrer?

Reply:
I don't know, it seems clear to me.